source/System.cpp

```diff
@@ -167,7 +167,7 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 				// that they are no longer here.
 				for(StellarObject &object : objects)
 					if(object.GetPlanet())
-						planets.Get(object.GetPlanet()->TrueName())->RemoveSystem(this);
+						object.GetPlanet()->RemoveSystem(this);
```

Comment:
system is already friend, do we need a getter?

Reply:
`planet` was a const pointer. I suppose I could leave it as non-const member without a non-const getter, but I'm not sure if that's a better solution.

Reply:
If the non-const getter is used only here, in a place where we can access private members directly, I don't think it's necessary

Reply:
agreed with tom